GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs

```diff
@@ -361,18 +361,38 @@ private void OnGuiLoadingScreenSelect()
             EditorGUILayout.EndHorizontal();
             EditorGUILayout.Space();
 
-            if (GUILayout.Button("Create Loading Scene"))
+            if (LoadingScreenGenerator.LoadingScreenExists())
```

Comment:
Instead of changing the code, can we have the same code but just change the button text based on LoadingScreenGenerator.LoadingScreenExists() ?

Reply:
One generates the scene and one just updates the image. Would you want to move the if statement to the inside of the button and have text change based on the existence of the loading screen?

Reply:
Resolved offline! Keeping it the same.